Makefile

```diff
@@ -30,6 +30,8 @@ build:
 # Use `make checkApi TAG=ReleaseTag` to check the release tag or `make checkApi` to check current branch while running locally.
 .PHONY: checkApi
 checkApi:
+	unset GITHUB_TOKEN; \
+	echo "$(shell mbx-ci github writer private token)" > gh_token.txt;\
```

Comment:
Won't it create untracked gh_token in the current dir?

Reply:
Yes, already added it into git ignore file before.